index.d.ts

```diff
@@ -0,0 +1,4 @@
+
+export function Encode(input: ArrayBuffer): ArrayBuffer
```

Comment:
Encode allows an optional second parameter to only look at a prefix of the overall buffer. is it worth representing that here?

Reply:
ok, I'll take a look, it's possible

Reply:
@willscott done